shap/explainers/_deep/__init__.py

```diff
@@ -126,7 +126,7 @@ def shap_values(self, X, ranked_outputs=None, output_rank_order='max', check_add
             ranked_outputs, and indexes is a matrix that indicates for each sample which output indexes
             were chosen as "top".
 
-           .. versionchanged:: 0.45.0.
+           .. versionchanged:: 0.45.0
            Return type for models with multiple outputs and one input changed from list to np.ndarray.
```

Comment:
I believe RST directives need to be indented like so:

```suggestion
           .. versionchanged:: 0.45.0
              Return type for models with multiple outputs and one input changed from list to np.ndarray.
```